docs/OPERATOR_HUB.md

```diff
@@ -34,7 +34,7 @@ Follow the steps:
 
 5. Copy the bundle directory to the community-operators repository directory. On the example below I got the community-operators repository cloned to `../../../github.com/k8s-operatorhub/community-operators`: 
    ```shell
-   cp -r bundle ../../../github.com/k8s-operatorhub/community-operators/operators/cc-operator/${TARGET_RELEASE}
+   cp -r bundle/* ../../../github.com/k8s-operatorhub/community-operators/operators/cc-operator/${TARGET_RELEASE}
```

Comment:
Well the old one works well if the target dir does not exists. The new one only works when the target dir exists. So I'd suggest either keeping the old one but adding `rm -rf ../../../github.com/k8s-operatorhub/community-operators/operators/cc-operator/${TARGET_RELEASE}` (which I favour as we ensure all previous files are cleaned) or the new-one but adding `mkdir -p ../../../github.com/k8s-operatorhub/community-operators/operators/cc-operator/${TARGET_RELEASE}`.

Alternatively we could use rsync, which is slightly more predictable ;-)

Reply:
(another benefit of `rm` is that it would complain if the parent dir won't exists (eg. someone uses incorrect number of `../`) while `mkdir -p` simply creates everything...)